tests/escape-args.rs

```diff
@@ -16,7 +16,7 @@ fn escape_args() {
     // escaping.
     for &arg in &["x", "", " ", "  ",
                   r" \ ", r" \\ ", r" \\\ ",
-                  r#"""#, r#""""#, r#"\"\\""#,
+                  r#"""#, r#"\"\\""#,
```

Comment:
Just a quick note of explanation here:

```
 ./target/debug//just-echo.exe ""
```

Outputs a blank, which doesn't match "".  Previously, it was escaping these quotes and matching via the escaped quotes.